news/2017-09-15-mapping-with-rewards-returns.md

```diff
@@ -0,0 +1,83 @@
++ [4:13 uncut, pre-timed .osz](https://assets.ppy.sh/artists/15/osz/Fractal%20Dreamers%20-%20Fortuna%20Redux.osz)
++ [2:47 MWR-cut, pre-timed .osz](https://assets.ppy.sh/artists/15/mwr/Fractal%20Dreamers%20-%20Fortuna%20Redux%20%28MWR%20version%29.osz)
+
+### \*namirin - closing eyes
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/18/previews/351.mp3" type="audio/mpeg">
+</audio>
+
+> From [\*namirin's](https://osu.ppy.sh/beatmaps/artists/18) latest album, 5colors. Bright vocals without excessive sweetness, set to a backing track that builds on victorious overtones. Likely familiar to anyone who is fond of anime.
++ [4:08 uncut, pre-timed .osz](https://assets.ppy.sh/artists/18/osz/namirin%20-%20closing%20eyes.osz) 
++ [2:37 MWR-cut, pre-timed .osz](https://assets.ppy.sh/artists/18/mwr/namirin%20-%20closing%20eyes%20(MWR%20version).osz)
+
+### Loki - A New King Is Born
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/7/previews/71.mp3" type="audio/mpeg">
+</audio>
+
+> The cinematic climax of [Loki/Thaehan's](https://osu.ppy.sh/beatmaps/artists/7) *The Fall of a Hero*. High BPM symphonic metal with prominent electric guitar melodies and leads, set to muted but varied percussion.
++ [3:22 uncut, pre-timed .osz](https://assets.ppy.sh/artists/7/osz/Loki%20-%20A%20New%20King%20Is%20Born.osz)
+
+## Bounty
+The bounty is claimable **once per gamemode per song** - for the sake of simplicity all of osu!mania's keycounts are treated as one gamemode.
+
+If your submission is judged to be the best map of the lot, here's the bounty we set on this contest:
+
+- 6 months of osu!supporter for you or your friends
+- Unique, mode specific Mapping With Rewards 2017 profile badge
+- Winning entries will be fast-tracked to Ranked via the dedicated attention of a selection of [Beatmap Nominators](https://osu.ppy.sh/groups/28) and [Quality Assurance Team](https://osu.ppy.sh/groups/7) members
+- Winning entries will receive special placement on osu! granting more visibility than normal mapsets
+
+## Conditions
+- **Submissions must be entered via the [contest system](https://osu.ppy.sh/community/contests/52).** The beatmap sets may also be submitted via the [Beatmap Submission System](/wiki/Submission), modded on the page and even reach Ranked status while this contest is running. You can submit multiple entries on this page if you wish to enter into more than one song. Maps only uploaded via the Beatmap Submission System will not be considered.
+- **Entries may be submitted until 31. December 2017.**  
+- **Submissions must adhere to the [Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria)** or be in a state close to satisfying this condition through minor changes.
+- **Custom hitsound samples cannot be used.** 
+- **Collaborations are not allowed because we aim for coherent sets**.
```

Comment:
No point in justifying (because blabla). There's no justification for the rest.

Reply:
there's a point, i don't want to be asked about this while i dont mind the questions about the rest